src/classes/HH_Households_TDTM.cls

```diff
@@ -239,7 +239,7 @@ public without sharing class HH_Households_TDTM extends TDTM_Runnable {
             
             if (householdnameupdates.size() > 0){
                               
-                if (currentHouseholdsSettings.npo02__Async_Household_Naming__c == true)
+                if (currentHouseholdsSettings.npo02__Async_Household_Naming__c == true && !system.isFuture() && !system.isBatch())
```

Comment:
Is this to fix a bug unrelated to the manage households UI?  If so, do you think it is better to apply it directly to dev in a separate PR, rather than tie it up with the UI work?